### PR TITLE
Fix typo in workflow xml that was causing corrupt post function

### PR DESCRIPTION
**Pull request by nickfyson**
This change to the XML fixes the observed problem with the post-functions on the 'create' transition.

FYI @aibaars @Daverlo

### CODE REVIEW

File: src/main/resources/workflow/workflow.xml [hunk 1]
```diff
@@ -23,9 +23,8 @@
               <arg name="class.name">com.atlassian.jira.workflow.function.issue.IssueReindexFunction</arg>
             </function>
             <function type="class">
-              <arg name="eventTypeId">1</arg>
-              <arg name="class.name">com.atlassian.jira.workflow.function.event.FireIssueEventFunction
-                            </arg>
```
Comment (aibaars) [inline src/main/resources/workflow/workflow.xml]: wouldn't the problem be the newline?

Comment (nickfyson) [reply src/main/resources/workflow/workflow.xml]: I wasn't sure. I guessed that the `eventTypeId` should be the same as for the other transitions in the file?

Comment (aibaars) [reply src/main/resources/workflow/workflow.xml]: `1` is `Issue Created` and `13` is `Generic Event`

Comment (nickfyson) [reply src/main/resources/workflow/workflow.xml]: Yeah, sorry, just looked properly at that Slack message...!

**Comment by aibaars** [comment]
Thanks!

**Pull request merged**